assets/stylesheets/pure-menu.css

```diff
@@ -0,0 +1,197 @@
+/*!
+Pure v0.6.0
+Copyright 2014 Yahoo! Inc. All rights reserved.
+Licensed under the BSD License.
+https://github.com/yahoo/pure/blob/master/LICENSE.md
+*/
```

Comment:
because the module brings the entire framework, 
we just need a few classes, 
and it is modified
https://www.npmjs.com/package/purecss